.github/workflows/integ-test-inject-develocity-short-lived-tokens.yml

```diff
@@ -0,0 +1,151 @@
+name: Test develocity injection
```

Comment:
Instead of adding a separate integration test, let's merge this coverage into the existing `inject-develocity` test. So we'll be testing develocity injection, and verifying that a short-lived token is used.